### PR TITLE
Added new Pet Types to `Pets.tsx`

### PR DESCRIPTION
**Pull request by siddhantprateek**
Issue #451 
- Added Pet SVG icons to the PetIcon folder

### CODE REVIEW

File: src/components/pets/Pets.tsx [hunk 1]
```diff
@@ -20,6 +24,30 @@ type PetLink = {
...
+    type: "scales-fins-other",
+    img: ScalesFinIcon,
+  },
+  {
+    type: "barnyard",
+    img: BarnyardIcon,
+  },
+];
+// eslint-disable-next-line no-console
+console.log(typeData);
```
Comment (redxzeta) [inline src/components/pets/Pets.tsx:49]: remove the consolelog

Comment (siddhantprateek) [reply src/components/pets/Pets.tsx]: done

File: src/components/pets/Pets.tsx [hunk 2]
```diff
@@ -20,6 +24,28 @@ type PetLink = {
...
+  {
+    type: "scales-fins-other",
+    img: ScalesFinIcon,
+  },
+  {
+    type: "barnyard",
+    img: BarnyardIcon,
+  },
+];
+
```
Comment (redxzeta) [inline src/components/pets/Pets.tsx:48]: why is this here?

**Review by redxzeta — CHANGES_REQUESTED** [review]
add the new types in this file `src\types\PetType.ts`

**Comment by redxzeta** [comment]
@siddhantprateek are you going to fix?

**Pull request closed without merging**